pycaret/containers/models/time_series.py

```diff
@@ -9,16 +9,17 @@
 # to complete the process. Refer to the existing classes for examples.
 
 import logging
-from typing import Union, Dict, Any, Optional
+from typing import Union, Dict, List, Tuple, Any, Optional
 
-import numpy as np
+import numpy as np  # type: ignore
+import random
 
-from sktime.forecasting.base._sktime import _SktimeForecaster
-from sktime.forecasting.compose import ReducedForecaster, TransformedTargetForecaster
-from sktime.forecasting.trend import PolynomialTrendForecaster
-from sktime.transformations.series.detrend import Deseasonalizer, Detrender
-from sktime.forecasting.base._sktime import DEFAULT_ALPHA
-from sklearn.ensemble import RandomForestRegressor
+from sktime.forecasting.base._sktime import _SktimeForecaster  # type: ignore
```

Comment:
which kind of errors does mypy warns here ?

Reply:
It checks for type annotations but lot of these base libraries were built before type annotations were introduced in Python (or do not have type annotations in them) so it says that it can not check types.